src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is started!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator is stopped!");
```

Comment:
Similarly, the message 'Excavator is stopped!' could be more informative. Consider rephrasing it to something like 'Excavator has stopped working.' to provide clearer information.